packages/canary-integration/src/__tests__/capability.test.ts

```diff
@@ -78,6 +81,64 @@ test('verifies capability with signed commit', async () => {
   expect(deterministicDocument.content).toEqual({ foo: 'bar' })
 }, 30000)
 
+test('does not allow updating if cacao issuer is not document controller', async () => {
+  // Create a did:pkh for the user
+  const wallet = Wallet.fromMnemonic(
+    'despair voyage estate pizza main slice acquire mesh polar short desk lyrics'
+  )
+  const didPkh = new DID({ resolver: PkhDidResolver.getResolver() })
+  // Create a determinstic tiledocument owned by the user
+  const deterministicDocument = await TileDocument.deterministic(
+    ceramic,
+    {
+      deterministic: true,
+      family: 'testCapabilities',
+    },
+    {
+      asDID: didPkh,
+    }
+  )
+
+  const streamId = deterministicDocument.id
+
+  // Create did:key for the dApp
+  const seed = new Uint8Array([
+    69, 90, 79, 1, 19, 168, 234, 177, 16, 163, 37, 8, 233, 244, 36, 102, 130, 190, 102, 10, 239, 51,
+    191, 199, 40, 13, 2, 63, 94, 119, 183, 225,
+  ])
+  const didKeyProvider = new Ed25519Provider(seed)
+  const didKey = new DID({ provider: didKeyProvider, resolver: KeyDidResolver.getResolver() })
+  await didKey.authenticate()
+
+  // Create CACAO with did:key as aud
+  const siweMessage = new SiweMessage({
+    domain: 'service.org',
+    address: wallet.address,
+    chainId: '1',
+    statement: 'I accept the ServiceOrg Terms of Service: https://service.org/tos',
+    uri: didKey.id,
+    version: '1',
+    nonce: '23423423',
+    issuedAt: new Date().toISOString(),
+    resources: [`ceramic://${streamId.toString()}`],
+  })
+  // Sign CACAO with did:pkh
+  const signature = await wallet.signMessage(siweMessage.toMessage())
+  siweMessage.signature = signature
+  const capability = Cacao.fromSiweMessage(siweMessage)
+  // Create new did:key with capability attached
+  const didKeyWithCapability = didKey.withCapability(capability)
+  await didKeyWithCapability.authenticate()
+
+  await expect(
+    deterministicDocument.update({ foo: 'baz' }, null, {
+      asDID: didKeyWithCapability,
+      anchor: false,
+      publish: false,
+    })
+  ).rejects.toThrow()
```

Comment:
```suggestion
  ).rejects.toThrow(/something something/)
```
Can we include simple regex so that we see that it throws where we expect it to throw?